ztp/source-crs/SriovFecSubscriptionNS.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vran-acceleration-operators
+  annotations:
+    workload.openshift.io/allowed: management
+  labels:
+    openshift.io/run-level: "1"
```

Comment:
This is a very dangerous label, from a security perspective.  Read more here: https://cloud.redhat.com/blog/exactly-what-are-openshift-runlevels-and-should-you-really-use-them

Reply:
Solely based on the sriov subscription here:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/SriovSubscriptionNS.yaml#L8

Reply:
@serngawy con you confirm if this is fully necessary? @lack is concerned about the security in this template